packages/engine-server/src/DendronEngineV3.ts

```diff
@@ -910,11 +913,47 @@ export class DendronEngineV3 extends EngineV3Base implements DEngine {
   private async initNotes(
     schemas: SchemaModuleDict
   ): Promise<RespWithOptError<NotePropsByIdDict>> {
-    const ctx = "DEngine:initNotes";
-    this.logger.info({ ctx, msg: "enter" });
+    const ctx = "DEngineV3:initNotes";
     let errors: IDendronError[] = [];
     let notesFname: NotePropsByFnameDict = {};
     const start = process.hrtime();
+    const enableSQLITE =
```

Comment:
`DendronEngineV3` shouldn't be aware of anything related to SQLite - it should only know about the `IDataStore` interface, otherwise we're losing the benefits we gained by creating those interfaces. We need to pull all init logic out of this class/file.

Reply:
What happens if a user has the SQLite setting flipped in their configs and is using the web extension? wouldn't it cause problem at this point?